### PR TITLE
Testing atomic

### PR DESCRIPTION
**Pull request by petervo**
Because we are going to want to switch commits during tests actually install the packages we are testing into a tree. Also actually install the version of cockpit-ws we are testing into the container instead of just using the default container.

### CODE REVIEW

**Comment by dperpeet** [comment]
good job on cleaning up the package installation for atomic!

can we move the changes to the tests into a separate commit?

**Comment by petervo** [comment]
@dperpeet could you take a look at the atomic check-network failure, it seems related to the recent libvirt change. Though I haven't been able to reproduce locally.

**Comment by stefwalter** [comment]
TestKubernetes.testDashboard failure is #2914

**Comment by stefwalter** [comment]
TestShutdownRestart failure is #2915

File: test/check-dashboard [hunk 1]
```diff
@@ -188,9 +188,13 @@ class TestDashboard(MachineCase):
         m2.wait_boot()
 
         # Create some users on m1 and m2.
+        groups1 = m1.execute("groups")
+        if not "docker" in groups1:
```
Comment (stefwalter) [inline test/check-dashboard]: Where is this docker group coming from? Is this a regression in Atomic Host? I thought that Atomic Host shouldn't have a docker group by default.

File: tools/cockpit.spec [hunk 2]
```diff
@@ -427,8 +427,17 @@ cluster. Installed on the Kubernetes master. This package is not yet complete.
 
 %if %{defined gitcommit}
 
+%package playground
```
Comment (stefwalter) [inline tools/cockpit.spec]: Why are we splitting this out? Is it not possible to force install cockpit-test-assets?

**Comment by petervo** [comment]
Pulled in @dperpeet's commit to work around the issues with multiple uses of the same mac address for now.

This is ready for review, once it's reviewed i'll change the tags so it saves the new image.

**Comment by stefwalter** [comment]
Looks good to me. @petervo What tags need changing?

**Comment by petervo** [comment]
removed the clean:f22-atomic tag and pushed again so the new image will build.

**Pull request closed without merging**